src/web/lib/decidePalette.ts

```diff
@@ -144,9 +144,18 @@ const textArticleLinkHover = (format: Format): string => {
 
 const backgroundArticle = (format: Format): string => {
 	// Order matters. We want comment special report pieces to have the opinion background
-	if (format.design === Design.Comment) return opinion[800];
-	if (format.theme === Special.SpecialReport) return specialReport[800];
-	return 'transparent';
+	if (
```

Comment:
This whole bit still feels a bit (though mostly because the underlying logic is just confusing rather than the code itself). But what about:

    if (format.design === Design.Comment) return opinion[800];
    if (format.theme === Special.SpecialReport) return specialReport[800];
    if (format.design === Design.GuardianView) return opinion[800];
    return 'transparent';

(it's not a major thing either way but seems slightly clearer as only one type of control flow and also the code mirrors the actual priority order.)

Reply:
oh snap!

Reply:
Your code is _much_ cleaner. Adopted. (I reversed special and guardianview though as I think we want to prioritse guardianview over special)

Reply:
Sorry, Nic won here. I read top to bottom

Reply:
😭